src/components/fsr3/mod.rs

```diff
@@ -373,7 +371,8 @@ impl Context {
 
     /// # Safety
     ///
-    /// You shouldn't call dispatch on a Context that had destroy called upon.
+    /// User is responsible for ensuring that all [Resource] and [CommandList] outlive the lifetime of [Context].
+    /// Additionally synchronization is expected to be handled externally.
```

Comment:
Looks clear to me :) Maybe unrelated question: I vaguely remember there being problems with getting synchronization to work because we are using enhanced barriers whereas (I assume) FSR3 is not? Is that something that is still relevant, or was this solved already? This is something that came up over a year ago when this was being worked on.